server.js

```diff
@@ -16,9 +17,62 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+const HappyThoughtSchema= new mongoose.Schema({
+thought: {
+
+type: String,
+required:true,
+minlength:5,
+maxlength:140,
+trim: true
+},
+likes:{
+ type: Number,
+ default: 0
+},
+createdAt:{ 
+  type: Date,
+  default:()=>new Date()
```

Comment:
you can add something like this to get the right time 
` default: () =>
      new Date(Date.now()).toLocaleString("sv-SE", {
        timeZone: "Europe/Stockholm",
      }),`

Maybe modified to Berlin, or is it right for you?